### ISSUE TITLE
[Android] [Integrate] As a user, I can see the job information in the job details page

### ISSUE THREAD

**Issue by luongvo** [nimblehq/android-kmm-showcases#15]
## Why

As we're creating a demo to show job detail, the user should be able to read the details of a job, including price, location, time, etc before deciding to accept the jobs or not.

## Acceptance Criteria

- Request the job detail data when opening the job detail screen.
- Show the loading progress while requesting the data.
- Show the error toast or dialog in case of error.
- Bind the data from API response by:
1 + 9: `minimum_price`-`maximum_price` Baht
2: `confirmed_at` in `dd MMM yyyy`
3: `time_slot.time_slot_start_at`-`time_slot.time_slot_end_at`
4:  _skip for now, as we need to implement Google Geocoder to calculate distance from job location to the user location_
5: `plot.size` m2
6:  `plot.address`
7: `plot.crop_type`
8: `message`

## Design

![image](https://user-images.githubusercontent.com/16315358/159992896-878c830f-dbde-4411-a055-3e7ed3014698.png)

## Resources

N/A

**Comment by luongvo** [comment]
@hoangnguyen92dn I think we need to create a UiModel for data preparing before sending them into platform rendering 🤔

**Comment by hoangnguyen92dn** [comment]
@luongvo Yup...I intend to do that in the integration task 👍  Since we don't have ViewModel so I will create UseCase and map to UiModel then.
